datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -71,34 +91,122 @@ Where:
 * `--existing-replica-id` is the id of the replica to backup,
 * `--ucp-username`, and `--ucp-password` are the credentials of a UCP administrator.
 
+To avoid having to pass the password as a command line parameter, you may
+instead use the following approach in bash:
+
+```none
+$ read -sp 'ucp password: ' PASS; UCP_PASSWORD=$PASS docker run -i --rm -e UCP_PASSWORD docker/dtr backup \
+  --ucp-url <ucp-url> \
+  --ucp-insecure-tls \
+  --existing-replica-id <replica-id> \
+  --ucp-username <ucp-admin> > /tmp/backup.tar
+```
+
+This puts the password into a shell variable which is then passed into the
+docker client command with the -e flag which in turn relays the password to the
+DTR bootstrapper.
+
+## Testing backups
+
 To validate that the backup was correctly performed, you can print the contents
 of the tar file created:
 
 ```none
 $ tar -tf /tmp/backup.tar
 ```
 
+The structure of the archive should look something like this:
+
+```none
```

Comment:
Can you can this sample to be just 4-6 lines. Right now we don't have vertical scroll 😭
So it will take a lot of space in the page and make it look strange.